extensions/shared/upgrade-nudge/style.scss

```diff
@@ -0,0 +1,44 @@
+.paid-block__wrapper {
+	border: 1px solid var( --color-border-subtle );
+}
+
+.paid-block__disabled {
+	opacity: 0.5;
+	padding: 10px;
+}
+
+.upgrade-nudge {
```

Comment:
Many of these class names are kinda generic and could collide with other plugins. How about we prefix with `[class^="wp-block-jetpack-"]`? Or just `jetpack_` in each?

Reply:
Thought this could cause a need for another `wp-block-a8c-` prefix if/when there are paid blocks which are not shipped in Jetpack...

Reply:
This is generic enough a component that I hope it'll make it into `@automattic/calypso-ui` eventually, with the style prefix....TBD :grimacing: 

For now, I guess prefixing with either `[class^="wp-block-jetpack-"]` or `jetpack_` is fine :+1:

Reply:
Can't do `[class^="wp-block-jetpack-"]` since we're wrapping _around_ that level of `<div />`.